src/ButtonGroup/index.js

```diff
@@ -0,0 +1,149 @@
+import React, { PureComponent, Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { transparentize } from 'polished';
+
+import Button from '../Button';
+
+const { func, node, string } = PropTypes;
+
+/**
+ * Button Group
+ *
+ * This component is in charge of displaying
+ * a button group
+ *
+ * @param {node} children // Anything that can be rendered: numbers, strings, elements or an array (or fragment).
+ * @param {string} className // Add a text aside in the select next the selected value.
+ * @param {function} onChange // Callback function called when a button is clicked.
+ * @param {string} defaultActiveButton // Specifies the initial state: witch button is selected.
+ *
+ * @return {jsx}
+ */
+
+class ButtonGroup extends PureComponent {
+  /**
+   * PropTypes Validation.
+   */
+  static propTypes = {
+    children: node,
+    className: string,
+    onChange: func,
+    defaultActiveButton: string,
+  };
+
+  /**
+   * Default props.
+   */
+  static defaultProps = {
+    children: null,
+    className: '',
+    onChange: () => {},
+    defaultActiveButton: '',
+  };
+
+  /** Internal state. */
+  state = {
+    activeButton: this.props.defaultActiveButton, // eslint-disable-line react/destructuring-assignment
```

Comment:
Okay... that's because I already wrote other comments... but this slice of the state holds... the name of the active button no? Maybe you could guess what name I'd use :-).